composer.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "humanmade/core-sitemaps",
+  "description": "Core Sitemaps project",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0-or-later",
+  "config": {
+    "preferred-install": "dist",
+    "sort-packages": true,
+    "autoloader-suffix": "csm",
+    "platform": {
+      "php": "7.3"
+    },
+    "process-timeout": 600,
+    "vendor-dir": "vendor"
+  },
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://wpackagist.org"
+    }
+  ],
+  "scripts": {
+    "post-install-cmd": [
+      "@setup"
+    ],
+    "post-update-cmd": [
+      "@setup"
+    ],
+    "setup": [
+      "@composer run-script --list"
+    ],
+    "local-tests": [
+      "@test:phpunit-local",
+      "@test:behat-local"
+    ],
+    "test:phpunit": [
+      "vendor/bin/phpunit --verbose --colors=always"
+    ],
+    "test:phpunit-local": [
+      "wp @local db query --dbuser=root --dbpass=password \"CREATE DATABASE IF NOT EXISTS wordpress_test DEFAULT CHARSET utf8\"",
+      "vagrant ssh -c \"cd /vagrant/content/plugins/core-sitemaps && WP_TESTS_DB_PASS=password composer run test:phpunit\""
+    ],
+    "test:phpcs": [
+      "vendor/bin/phpcs -nps --colors --report-code --report-summary --report-width=80 ."
+    ],
+    "test:behat-local": [
+      "# Make sure Selenium is running!",
+      "BEHAT_PARAMS='{\"extensions\":{\"Behat\\\\MinkExtension\":{\"base_url\":\"http://sitemaps.local\"},\"PaulGibbs\\\\WordpressBehatExtension\":{\"site_url\":\"http://sitemaps.local/wp\",\"wpcli\":{\"alias\":\"local\"}}}}' vendor/bin/behat --colors --format progress --"
+    ],
+    "test:behat-steps": [
+      "BEHAT_PARAMS='{\"extensions\":{\"Behat\\\\MinkExtension\":{\"base_url\":\"http://sitemaps.local\"},\"PaulGibbs\\\\WordpressBehatExtension\":{\"site_url\":\"http://sitemaps.local/wp\",\"wpcli\":{\"alias\":\"local\"}}}}' vendor/bin/behat --colors --definitions i"
+    ]
+  },
+  "scripts-descriptions": {
+    "setup": "Sets up the development environment.",
+    "local-tests": "Runs all tests (PHPUnit and Behat) -- for local environments.",
+    "test:phpunit": "Run PHPUnit tests.",
+    "test:phpunit-local": "Run PHPUnit tests -- for local environments.",
+    "test:phpcs": "Runs the PHP code sniffer.",
+    "test:behat-local": "Run Behat tests -- for local environments.",
+    "test:behat-steps": "View step definitions for Behat tests -- for local environments."
+  },
+  "require": {
+    "php": "^5.6.0||^7.0",
+    "composer/installers": "~1.0",
+    "oomphinc/composer-installers-extender": "^1.1"
+  },
+  "require-dev": {
+    "roave/security-advisories": "dev-master",
+    "behat/mink-goutte-driver": "^1.2",
+    "behat/mink-selenium2-driver": "^1.3",
+    "genesis/behat-fail-aid": "^2.0",
+    "humanmade/coding-standards": "^0.3",
```

Comment:
I think this needs to be the WordPress coding standards rather than Humanmade. This is something I can update as part of https://github.com/humanmade/core-sitemaps/pull/14 though as it also affects the Travis CI builds

Reply:
Ok happy with that @kirstyburgoine as that will avoid any mistakes from adding it in twice.